index.js

```diff
@@ -1,14 +1,37 @@
+const {
+  fetchJSON,
+  getOwner,
+  getRepo
+} = require('./util')
 /**
  * This is the entry point for your Probot App.
  * @param {import('probot').Application} app - Probot's Application class.
  */
+var base64 = require('base-64')
```

Comment:
This looks ancient (yet working though). Still, would suggest the current [popular](https://www.npmjs.com/package/js-base64) one. Works just the same, and is lightweight.